plugin-server/src/main/services/http-server.ts

```diff
@@ -29,6 +29,12 @@ export function createHttpServer(hub: Hub, serverInstance: ServerInstance, serve
                 }
             }
 
+            // Unlike the above healthcheck, this is more of a "readiness" check that verifies that the consumer
```

Comment:
That's a mess :sweat_smile:  - could we prioritize cleaning up the healthcheck/readiness story up and providing proper names to things?

Reply:
The problem is that ECS doesn't have readiness checks, hence this mess of concerns

Reply:
We'll be on k8s hopefully really soon \o/